### DISCUSSION TITLE
How would I configure qbit-manage to run with 2 instances of qbittorrent via docker compose?

### DISCUSSION THREAD

**Discussion by pinkairfreshener** [StuffAnThings/qbit_manage#600] [Q&A]
I have qbittorrent-movies and qbittorrent-tv which work well and each has thousands of torrents. Looking at the compose file sample it is asking for qbittorrent_Backup folder and torrent download dir etc however which do I put in since I have 2 qbittorrents running?
Thanks

**Comment by nosirrahdrof** [comment]
Curious if you ever found a solution to this?   I am looking to manage 3 qbit instances
